src/main/kotlin/blackjack/model/Participant.kt

```diff
@@ -0,0 +1,90 @@
+package blackjack.model
+
+sealed class Participant(private val name: ParticipantName, private val hand: Hand) {
+    fun getName() = name
+
+    fun getCards() = hand.getCards()
+
+    fun getCardsSum() = hand.calculateCardsSum()
+
+    fun drawCard(card: Card) {
+        hand.addCard(card)
+    }
+
+    fun getState() = hand.calculateState()
+
+    fun isGameFinished() = getState() == State.Blackjack || getState() == State.Bust
```

Comment:
```suggestion
    private fun getState() = hand.calculateState()

    private fun isGameFinished() = getState() == State.Blackjack || getState() == State.Bust
```

> Participant 내에서 Hand를 사용하는 여러 메서드를 추가됐고 많은 get 메서드도 생겼는데, 제가 보기에는 좀 지저분해보이는데.. 과연 이러한 구조가 괜찮은지 모르겠습니다 ㅠ

현재 Participant 에 새로 추가된 메서드들이 Participant 에서만 쓰이고 있는데, 굳이 public 하게 공개한 이유가 궁금합니다.
DealerTest 와 PlayerTest 를 위함인가요?
만약 그렇다면 Hand 에 대한 테스트를 강화하는 방식으로 해도 괜찮지 않을까 하는 생각을 했습니다.
Participant 의 역할을 Hand 로 옮기는 리팩토링을 해주신 거니까요.

아니면 굳이 getter 메서드를 추가하지 말고, hand 맴버에 직접 접근해서 `calculateState()` 를 호출하는 것도 한 방법인 것 같습니다.